### PR TITLE
Ftr/direct samples

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: direct/go-client/cmd/client.go [hunk 1]
```diff
@@ -19,51 +19,36 @@ package main
...
+	}
+	reply, err := grpcGreeterImpl.SayHello(context.Background(), req)
 	if err != nil {
-		gxlog.CError("error: %v\n", err)
-		os.Exit(1)
-		return
+		logger.Error(err)
 	}
-	gxlog.CInfo("response result: %v\n", user)
+	logger.Infof("client response result: %v\n", reply)
```
Comment (xavier-niu) [inline direct/go-client/cmd/client.go:53]: Is this `\n` useful?

File: direct/go-client/conf/dubbogo.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dubbo:
+  consumer:
+    references:
+      "greeterImpl":
+        url: tri://localhost:20000
+        protocol: "tri"
+        interface: "com.apache.dubbo.sample.basic.IGreeter" # must be compatible with grpc or dubbo-java
```
Comment (xavier-niu) [inline direct/go-client/conf/dubbogo.yml:7]: Quotations should be deleted in yml files

File: direct/go-server/conf/dubbogo.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+dubbo:
+  protocols:
+    "triple":
+      name: "tri"
+      port: 20000
+  provider:
+    services:
+      "greeterImpl":
+        protocol: "triple"
+        interface: "com.apache.dubbo.sample.basic.IGreeter" # must be compatible with grpc or dubbo-java
```
Comment (xavier-niu) [inline direct/go-server/conf/dubbogo.yml:10]: quotations

File: integrate_test/rpc/codec-extension/conf/dubbogo.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    "triple":
+      name: "tri"
+      port: 20000
+  provider:
+    registry:
+      - demoZK
+    services:
+      "greeterImpl":
+        protocol: "triple"
+        interface: "com.apache.dubbo.sample.basic.IGreeter" # must be compatible with grpc or dubbo-java
```
Comment (xavier-niu) [inline integrate_test/rpc/codec-extension/conf/dubbogo.yml:17]: quotations

**Comment by LaurenceLiZhixin** [comment]
@xavier-niu Great comments,  I have fixed.

**Review by zhaoyunxing92 — APPROVED** [review]

**Review by xavier-niu — APPROVED** [review]
LGTM

**Pull request merged**